doc/about.rst

```diff
@@ -0,0 +1,123 @@
+.. _about:
+
+##############################################################################
+About
+##############################################################################
```

Comment:
I see. Then, I'll keep the long headers for now. It's detail we can always refactor later, eg. if @lexicalunit  prefers short headers too.